lib/home/profile/account/change_password.dart

```diff
@@ -1,17 +1,219 @@
 import 'package:flutter/material.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:thc/utils/navigator.dart';
 
-class ChangePasswordScreen extends StatelessWidget {
+class ChangePasswordScreen extends StatefulWidget {
   const ChangePasswordScreen({super.key});
 
+  @override
+  // ignore: library_private_types_in_public_api
+  _ChangePasswordScreenState createState() => _ChangePasswordScreenState();
+}
+
+class _ChangePasswordScreenState extends State<ChangePasswordScreen> {
+  final TextEditingController _currentPasswordController =
+      TextEditingController();
+  final TextEditingController _newPasswordController = TextEditingController();
+  final TextEditingController _confirmPasswordController =
+      TextEditingController();
+  bool _isCurrentPasswordCorrect = false;
+  bool _isCurrentPasswordVerified = false;
+  String _verificationMessage = '';
+  Color _messageColor = Colors.black;
```

Comment:
```suggestion
  Color? _messageColor;
```

If we set the color to black, it doesn't show up super well if the app is in dark mode.